wafer/kv/serializers.py

```diff
@@ -26,4 +75,13 @@ def __init__(self, *args, **kwargs):
         groups = self.fields['group']
         groups.queryset = user.groups
 
+        # Add fields for the models we have ManyToMany relationships defined for
+        for field in self.Meta.model._meta.get_fields():
+            if field.name not in self.fields and field.many_to_many:
+                # XXX: Is there a better way to get the correct view_name out of the url config?
```

Comment:
Unfortunately not - this needs to be the detail view name that rest_framework constructs, and that isn't a direct attribute of the model or the field.

Lookingat it again, rest_framework does provide a utility function which constructs this name, so I've switched to using that.